e2e/cypress/tests/integration/autocomplete/helpers.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {AdminConfig} from '@mattermost/types/lib/config';
+import {UserProfile} from '@mattermost/types/lib/users';
```

Comment:
These types were already under [Cypress namespace](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/index.d.ts#L7-L34)

we can use `Cypress.AdminConfig` etc

Reply:
thanks for the info @furqanmlk I will also use it for other PRs